Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.10
```

Comment:
Would you mind changing the first two lines to this:

```Dockerfile
FROM python:3.10-slim
RUN apt-get update --yes && apt-get install pipx python3-dev libpq-dev gcc --yes && pipx install poetry
```

and removing this line:

```Dockerfile
RUN poetry config virtualenvs.create false
```

Using the `python:3.10-slim` images makes the final image ~0.5 GB smaller (you can see image sizes by running `docker image ls` in the terminal). The `apt-get` commands are for install extra dependencies (the slim versions of docker images tend to have fewer libraries pre-installed), and `pipx` is to make it work better with the slim image (not 100% sure why, but `curl` just wasn't working with it).